### PR TITLE
update doxx

### PR DESCRIPTION
**Pull request by emirotin**
https://github.com/resin-io-projects/doxx/blob/master/CHANGELOG.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,23 +20,13 @@
   },
   "homepage": "https://github.com/resin-io/docs",
   "dependencies": {
-    "@resin.io/doxx": "^0.1.0",
+    "@resin.io/doxx": "^0.8.0",
     "bower": "^1.7.9",
     "coffee-script": "^1.10.0",
-    "consolidate": "^0.14.0",
```
Comment (emirotin) [inline package.json]: all removed deps are now sub-deps of doxx

File: pages/runtime/supervisor-api.md [hunk 2]
```diff
@@ -484,3 +484,142 @@ $ curl -X POST --header "Content-Type:application/json" \
 	--data '{"deviceId": <deviceId>, "appId": <appId>, "method": "GET"}' \
```
Comment (emirotin) [inline pages/runtime/supervisor-api.md]: these files were automatically refetched from sources

File: pages/getting-started.md [hunk 3]
```diff
@@ -1,15 +1,12 @@
 ---
-title: Getting Started with {{ $device_details.name }} and {{ $language_details.name }}
+title: Getting Started with {{ $device.name }} and {{ $language.name }}
```
Comment (emirotin) [inline pages/getting-started.md]: the majority of change are dropping the `_details` part

File: pages/getting-started.md [hunk 4]
```diff
@@ -1,15 +1,12 @@
...
 layout: getting-started.html
 
-# this is dynamic page definition
-# hope it's self-explanatory
-dynamic_page:
-  axes: [ $device, $language ]
-  url: $device/$language/$baseUrl
-  partials_search: [ $device+$language, $device, $language, _default ]
-  switch_text: Getting Started with $device and $language
+dynamic:
```
Comment (emirotin) [inline pages/getting-started.md]: the `dynamic` part was changed towards simiplification

File: pages/getting-started.md (hunk 3 above)
Comment (shaunmulligan) [inline pages/getting-started.md]: @emirotin do you want me to PR a change for this? on top of this PR?

Comment (shaunmulligan) [inline pages/getting-started.md]: nevermind, i just saw that you did it already...haha...sorry its early 👍

**Comment by shaunmulligan** [comment]
LGTM

**Pull request merged**